VERSION

```diff
@@ -1 +1 @@
-latest
```

Comment:
The main branch should have latest flag, while the release branch should have v0.5.0

Reply:
Updated to `latest` in the second commit